exercises/practice/kindergarten-garden/.meta/example.h

```diff
@@ -0,0 +1,8 @@
+#pragma once
+
+#include <array>
+#include <string>
+
+namespace kindergarten_garden {
+    std::array<std::string, 4> plants(std::string diagram, std::string student);
```

Comment:
In case you don't like the `std::string_view`, maybe pass the arguments via const reference at least?

Reply:
I do like string views. A lot in fact. But catch2 is not a fan and sometimes fails. I'll look into it again. Const ref would be better